decidim-core/app/helpers/decidim/user_profile_helper.rb

```diff
@@ -14,9 +14,13 @@ module UserProfileHelper
     #
     # Returns a String with the menu tab.
     def user_profile_tab(text, link, options = {})
```

Comment:
Note that this helper does not seem to be used anywhere in 0.28 but this change is needed in order to backport this fix to 0.27.

It is not in the scope of this PR to remove this.

Reply:
I agree. We'll handle this removal in another PR.